<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_rosenblatt</class>
 <widget class="QDialog" name="Dialog_rosenblatt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_plot"/>
  </widget>
  <widget class="QPushButton" name="pushButton_clearGraph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>251</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Выбор приближения</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_method">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>211</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_methodParams"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_drawEstimate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
